### PR TITLE
Add ability to configure calico failsafe ports

### PR DESCRIPTION
**Pull request by jonaz**
Made FailsafeOutboundHostPorts and FailsafeInboundHostPorts configurable
to allow users to have custom network policy for example SSH (22)

Previously failsafe rules would always have opened 22 even if user had
custom GlobalNetworkPolicy applied on the node.

Fixes https://github.com/rancher/rke2/issues/921

### CODE REVIEW

File: packages/rke2-canal/charts/values.yaml
Comment (Oats87) [inline packages/rke2-canal/charts/values.yaml:75]: I believe this should probably be prefixed with a `felix` and formatted in `camelCase` to keep with chart convention.

File: packages/rke2-canal/charts/values.yaml
Comment (Oats87) [inline packages/rke2-canal/charts/values.yaml:77]: I believe this should probably be prefixed with a `felix` and formatted in `camelCase` to keep with chart convention.

**Review by Oats87 — CHANGES_REQUESTED** [review]

File: packages/rke2-canal/charts/values.yaml
Comment (jonaz) [reply packages/rke2-canal/charts/values.yaml]: Fixed

File: packages/rke2-canal/charts/values.yaml
Comment (jonaz) [reply packages/rke2-canal/charts/values.yaml]: Fixed

**Review by Oats87 — APPROVED** [review]
LGTM

**Review by brandond — APPROVED** [review]

**Comment by jonaz** [comment]
@brandond do i need to do anything else? It says: `1 workflow awaiting approval, First-time contributors need a maintainer to approve running workflows. Learn more.
`

**Comment by brandond** [comment]
You need to also bump the version at https://github.com/rancher/rke2-charts/blob/main-source/packages/rke2-canal/package.yaml#L2

**Comment by jonaz** [comment]
> You need to also bump the version at
> 
> https://github.com/rancher/rke2-charts/blob/1b4dda478d884b21ee98637ea5e60673bfb3fe4c/packages/rke2-canal/package.yaml#L2

Just bumped it to 03. Hope that was correct :)

**Pull request merged**
